### ISSUE TITLE
`terraform_docs` starting from 1.94.0 ignores `--hook-config=--add-to-existing-file=false` (our default behavior)

### ISSUE THREAD

**Issue by MaxymVlasov** [antonbabenko/pre-commit-terraform#715]
### Describe the bug

```
terraform-docs --output-mode="inject" --output-file=README.md md .
```
which is exactly what we run by default 
https://github.com/antonbabenko/pre-commit-terraform/blob/91b5ba0c184ca6ef14cdca0b6d3266084dbba166/hooks/terraform_docs.sh#L279

by

```yaml
repos:
  - repo: https://github.com/antonbabenko/pre-commit-terraform
    rev: v1.94.2 # issue introduced in 1.94.0
    hooks:
      - id: terraform_docs
```

always updates/add `terraform-docs` section, as long as file exist.

This break your `--add-to-existing-file` setting, which by default set to `false`

### How can we reproduce it?

```bash
testdir=$(mktemp -d)
cd $testdir
touch main.tf README.md
echo "
repos:
  - repo: https://github.com/antonbabenko/pre-commit-terraform
    rev: v1.94.2 # issue introduced in 1.94.0
    hooks:
      - id: terraform_docs
" > .pre-commit-config.yaml
git init
git add -A
pre-commit run
```

### Workaround
* if you on Linux (and you have no Mac users) - stick to [v1.93.1](https://github.com/antonbabenko/pre-commit-terraform/releases/tag/v1.93.1)
* otherwise, stick to [v1.92.3](https://github.com/antonbabenko/pre-commit-terraform/releases/tag/v1.92.3)

**Comment by MaxymVlasov** [comment]
Workaround: 
* if you on Linux (and you have no Mac users) - stick on [v1.93.1](https://github.com/antonbabenko/pre-commit-terraform/releases/tag/v1.93.1)
* otherwise, stick to [v1.92.3](https://github.com/antonbabenko/pre-commit-terraform/releases/tag/v1.92.3)

**Comment by antonbabenko** [comment]
This issue has been resolved in version 1.94.3 :tada:
